src/index.njk

```diff
@@ -24,7 +28,7 @@ description: Design your service using GOV.UK styles, components and patterns
       </div>
 
       <div class="govuk-grid-column-full">
-        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
+        <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl">
```

Comment:
Is this change intentional? It seems to add more whitespace below the "Styles, Components, Patterns" cards:

| Before | After |
|-------|------|
| <img width="376" alt="Screenshot 2021-06-21 at 11 05 11" src="https://user-images.githubusercontent.com/503614/122745095-8f7e5480-d280-11eb-9f41-b16a56b385b6.png"> | <img width="378" alt="Screenshot 2021-06-21 at 11 05 33" src="https://user-images.githubusercontent.com/503614/122745152-9b6a1680-d280-11eb-9c3b-c2ee6f18c1ae.png"> |

Reply:
No that's not intentional :(

Reply:
No worries, I will take the change out 👍